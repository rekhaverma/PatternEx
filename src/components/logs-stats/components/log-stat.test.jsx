import React from 'react';
import LogStat from './log-stat.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<LogStat />', () => {
  it('should match with snapshot', () => {
    const props = {
      count: 1,
      label: 'testing logs',
    };
    const wrapper = shallow(<LogStat {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
