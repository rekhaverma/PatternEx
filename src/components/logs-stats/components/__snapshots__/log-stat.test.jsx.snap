// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogStat /> should match with snapshot 1`] = `
<div
  className="logs_stats__item"
>
  <span
    className="highlighted"
  >
    1
  </span>
  testing logs
</div>
`;
